package.json

```diff
@@ -178,10 +178,14 @@
 		"postinstall": "node ./scripts/postinstall.js",
 		"pretest": "npm run compile && npm run lint",
 		"lint": "eslint src --ext ts",
-		"test": "node ./out/test/runTest.js"
+		"test": "node ./out/test/runTest.js",
+		"test:ui:install": "node ./out/test/ui-test/launcher.js ./out/test/ui-test/install.test.js",
```

Comment:
what is a benefit to have two scripts `test:ui:install` and `ui-test`? I am just asking :)

Reply:
Nothing really. I have added ui-test goal because we use it in our pipelines. At first I wanted to separate tests into several goals. However I do not know if it will work because at this moment vscode-extension-tester has problems with running multiple testing instances.